src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -198,11 +202,15 @@ class LazyMobXTableStore<T> {
     @observable public sortColumn:string;
     @observable public sortAscending:boolean;
     @observable.ref public columns:Column<T>[];
-    @observable private _columnVisibility:{[columnId: string]: boolean};
     @observable public dataStore:IMobXApplicationDataStore<T>;
     @observable public downloadDataFetcher:IMobXApplicationLazyDownloadDataFetcher|undefined;
     @observable private highlightColor:string;
 
+    // this observable is intended to always refer to props.columnVisibility
+    @observable private _columnVisibility:{[columnId: string]: boolean}|undefined;
```

Comment:
not sure i understand.  if it always refers to props, why not just reference props directly and eliminate this?

Reply:
I also don't quite see where this gets set

Reply:
nevermind. i get it.  i forgot about that setProps thing.

Reply:
yea, this is a bit confusing, `props` is a local variable passed to `setProps`.